core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-field-dragabble-item/content-type-field-dragabble-item.component.ts

```diff
@@ -134,17 +143,13 @@ export class ContentTypesFieldDragabbleItemComponent implements OnInit, OnDestro
      */
     openAttr($event: MouseEvent) {
         $event.stopPropagation();
+        this.overlayPanel.show($event, $event.target);
         this.open = true;
-    }
 
-    /**
-     * This method is used to close the attributes list when the user clicks on the close button
-     * @param {MouseEvent} $event
-     * @memberof ContentTypesFieldDragabbleItemComponent
-     */
-    closeAttr($event: MouseEvent) {
-        $event.stopPropagation();
-        this.open = false;
+        setTimeout(() => {
+            this.overlayPanel.hide();
+            this.open = false;
+        }, 2000);
```

Comment:
Why do we need `setTimeout()`?

Reply:
Because the overlayPanel doesn't have a timeout to close and if the user keeps opening panels they won't close until they click somewhere else, so in that case you won't have a lot of panels open, because they will close after 2 seconds